### PR TITLE
don't upload strings that will later be copied from source (WP-904)

### PR DESCRIPTION
**Pull request by vsolovei-smartling**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -49,6 +51,7 @@ pipeline {
 
             steps {
                 archiveArtifacts artifacts: 'release.zip'
+                archiveArtifacts artifacts: '**/logfile-*'
```
Comment (vsolovei-smartling) [inline Jenkinsfile:54]: Helps when examining integration test results

File: Jenkinsfile [hunk 2]
```diff
@@ -36,8 +36,10 @@ pipeline {
             }
 
             steps {
-                withCredentials([string(credentialsId: 'PROJECT_ID', variable: 'PROJECT_ID'), string(credentialsId: 'USER_IDENTIFIER', variable: 'USER_IDENTIFIER'), string(credentialsId: 'TOKEN_SECRET', variable: 'TOKEN_SECRET')]) {
-                    sh 'docker run --rm -w /plugin-dir -v $PWD:/plugin-dir -e MYSQL_HOST=localhost -e CRE_PROJECT_ID=$PROJECT_ID -e CRE_USER_IDENTIFIER=$USER_IDENTIFIER -e CRE_TOKEN_SECRET="$TOKEN_SECRET" wordpress-localization-plugin-php74:latest'
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
```
Comment (vsolovei-smartling) [inline Jenkinsfile:39]: Added catchError to allow next step archive artifacts

File: tests/IntegrationTests/tests/AdvancedCustomFieldsTest.php [hunk 3]
```diff
@@ -222,7 +222,7 @@ public function testWP690AcfBlocksWithArraysDontCauseWarnings()
         $translationHelper = $this->getTranslationHelper();
         $sourceBlogId = 1;
         $targetBlogId = 2;
-        $postId = $this->createPost(ContentTypeHelper::CONTENT_TYPE_POST, 'title', file_get_contents(DIR_TESTDATA . '/wp-690-source.html'));
+        $postId = $this->createPost(ContentTypeHelper::CONTENT_TYPE_POST, 'WP-690', file_get_contents(DIR_TESTDATA . '/wp-690-source.html'));
```
Comment (PavelLoparev) [inline tests/IntegrationTests/tests/AdvancedCustomFieldsTest.php:225]: WP-690 - is not a related ticket. is it a mistake?

**Review by PavelLoparev — APPROVED** [review]

File: tests/IntegrationTests/tests/AdvancedCustomFieldsTest.php (hunk 3 above)
Comment (vsolovei-smartling) [reply tests/IntegrationTests/tests/AdvancedCustomFieldsTest.php:225]: No, this came up while testing, and to make debugging easier, I've introduced this change. Test is related to issue WP-690, and is commented in line 217

**Pull request merged**